debian/vassal.6

```diff
@@ -0,0 +1,206 @@
+.B \-c,\-\-chatlog
+Echo error log to chat log.
+.TP
+.BI \-e,\-\-edit \ module
+Edit the module in the file \fImodule\fP (typically ending in
+\fB.vmod\fP). 
+.TP
+.BI \-\-edit\-extension \ extension
+Edit the extension \fIextension\fP (typically ending in \fB.vmdx\fP or
+\fB.vext\fP).  Note that the corresponding module (typically ending in
+\fB.vmod\fP) \fImust\fP be known to Vassal.  That is, you \fImust\fP
+have the module in your module library by opening the module at least
+once, before opening its extension.
+.TP
+.BI \-i,\-\-import \ adc2\-gamebox
+Imports the \fIAide de Champ\fP (version 2) gamebox
+\fIadc2-gamebox\fP, and opens the editor or the newly created Vassal
+module.
+.TP
+.BI \-l,\-\-load \ module
+.TQ
+.BI \-l,\-\-load \ save
+.TQ
+.BI \-l,\-\-load \ log
+Open the module \fImodule\fP, game save \fIsave\fP, or play log
+\fIlog\fP in the Vassal player.
+.sp 1
+Note, when loading a game save (typically ending in \fB.vsav\fP) or
+play log (typically ending in \fB.vlog\fP), then the corresponding
+module (typically ending in \fB.vmod\fP) \fImust\fP be known to
+Vassal.  That is, you \fImust\fP have the module in your module
+library by opening the module at least once, before opening the save
+or log.
+.TP
+.B \-m,\-\-manage
+Opens up the module manager (default if no other option is specified).
+.TP
+.B \-n,\-\-new
+Create a new module. 
+.TP
+.BI \-\-new\-extension \ module
+Create a new module extension to \fImodule\fP. 
+.TP
+.B \-\-version
+Report the version of Vassal to standard output and exit.
+.PP
+Below are some options that are \fIspecific\fP to the Debian
+installation.
+.TP
+.BI \-D variable=value
+Pass the assignment of \fIvariable\fP to \fIvalue\fP to the underlying
+Java process.  See \fIjava\fP(1) for more.
+.TP
+.B \-g,\-\-debug
+Run the Vassal process in a debugger \fBjdb\fP(1). 
+.TP
+.BI \-s,\-\-source \ directory
+Set Vassal source directory for debugger.  This option can be
+specified multiple times, or \fIdirectory\fP can be a colon \fB:\fP
+separated list of directories.
+.TP
+.B \-\-direct
+By-pass the module manager and run player, editor, ..., directly
+(useful when debugging)
+.SH FILE TYPES
+.TP
+.IB module .vmod
+A module file.  These files contains the definition of a module, such
+as the board, pieces, cards, and so on.  The can also contained
+predefined scenarios, tutorials, and additional information.
+.TP
+.IB save .vsav
+Save of a game state.  These are snapshots of the game state at the
+time of the save.  These are useful for saving a game to resume from
+at a later time, or as definitions of scenarios.
+.TP
+.IB log .vlog
+Play log.  These contain the \fIfull\fP log (since the time of
+recording) of a play of a module.  This is typically what one would
+exchange in the PBEM play-mode.
+.TP
+.IB extension .vmdx
+A module extension.  Module extensions are similar to modules, but
+augment a module with more elements, such as new counters, boards,
+cards, and so on.
+.TP
+.IB extension .vext
```

Comment:
Ditto. Do not use or even mention `.vext`.

Reply:
Fixed